README.org

```diff
@@ -116,16 +116,16 @@
   - Proton
   - Lepton
 
-** F&A
+** FAQ
```

Comment:
TOC should be updated.

Reply:
@black7375 What does TOC mean? I edited F&A to FAQ because it’s more frequently used.

Reply:
TOC is Table Of Contents
https://github.com/black7375/Firefox-UI-Fix#lepton-old-name-proton-fix

`[[#fa][F&A]]` => `[[#faq][FAQ]]`

Reply:
OK. I’ll fix it right away.